ci/docker/base.ml.Dockerfile

```diff
@@ -5,13 +5,12 @@ FROM $DOCKER_IMAGE_BASE_TEST
 
 COPY . .
 
-RUN BUILD=1 ./ci/ci.sh init
-
 RUN <<EOF
 #!/bin/bash -i
 
 set -e
 
+BUILD=1 ./ci/ci.sh init
```

Comment:
move this into `bash -i` so it can find the conda installed in the base test

Reply:
fewer layers too~